README.md

```diff
@@ -195,52 +195,76 @@ Install custom reporters as a development dependency and you can use the `--repo
 nyc report --reporter=<custom-reporter-name>
 ```
 
-## Excluding files
+## Selecting files for coverage
 
-You can tell nyc to exclude specific files and directories by adding
-an `nyc.exclude` array to your `package.json`. Each element of
-the array is a glob pattern indicating which paths should be omitted.
+By default, nyc only collects coverage for source files that are visited during a test.
+It does this by watching for files that are `require()`'d during the test.
+Only source files that are visited during a test will appear in the coverage report and contribute to coverage statistics.  
 
-Globs are matched using [minimatch](https://www.npmjs.com/package/minimatch).
+Nyc will instrument all files if the `--all` flag is set.
+In this case all files will appear in the coverage report and contribute to coverage statistics.
 
-For example, the following config will exclude any files with the extension `.spec.js`,
-and anything in the `build` directory:
+Nyc will only cover files that are located under `cwd`, and then only `*.js` files or files with extensions listed in in the `nyc.extension` array.
 
-```json
-{
-  "nyc": {
-    "exclude": [
-      "**/*.spec.js",
-      "build"
-    ]
-  }
-}
+You can reduce the set of covered files by adding `include` and `exclude` filter arrays to your config.  
```

Comment:
I think the trailing spaces here are causing markdown to add `<br>` after this line and a few others.

Reply:
yep - I did not see them hence my confusion! Good eye!